### PR TITLE
fix: Fixes missing ordinal string comparisons arguments.

### PR DESCRIPTION
**Pull request by mdodkins**
Use overloads or alternative functions to resolve errors building in Linux Rider.

### CODE REVIEW

File: Projects/UOContent/Assistants/AssistantHandler.cs [hunk 1]
```diff
@@ -84,7 +84,7 @@ public static void AssistVersion(NetState state, CircularBufferReader reader, in
 
         // Instead we are supporting razor community edition.
         var assistVersion = reader.ReadAscii();
-        state.Assistant = assistVersion.Contains(' ') ? assistVersion : $"RazorCE {assistVersion}";
+        state.Assistant = assistVersion.Contains(' ', StringComparison.Ordinal) ? assistVersion : $"RazorCE {assistVersion}";
```
Comment (kamronbatman) [inline Projects/UOContent/Assistants/AssistantHandler.cs:87]: We should have a ContainsOrdinal convenience method?

Comment (kamronbatman) [inline Projects/UOContent/Assistants/AssistantHandler.cs:87]:
```suggestion
        state.Assistant = assistVersion.ContainsOrdinal(' ') ? assistVersion : $"RazorCE {assistVersion}";
```

File: Projects/UOContent/World Saves/AutoArchive.cs [hunk 2]
```diff
@@ -394,7 +394,7 @@ private static IEnumerable<string> PathsByTimestampName(string path, bool files
                 if (files)
                 {
                     var fileName = new FileInfo(item).Name;
-                    name = fileName[..fileName.IndexOf('.')];
+                    name = fileName[..fileName.IndexOf('.', StringComparison.Ordinal)];
```
Comment (kamronbatman) [inline Projects/UOContent/World Saves/AutoArchive.cs:397]:
```suggestion
                    name = fileName[..fileName.IndexOfOrdinal('.')];
```

**Review by kamronbatman — APPROVED** [review]

File: Projects/UOContent/Assistants/AssistantHandler.cs (hunk 1 above)
Comment (mdodkins) [reply Projects/UOContent/Assistants/AssistantHandler.cs]: Ahh nice

**Pull request merged**